vita3k/gui/src/home_screen.cpp

```diff
@@ -523,10 +523,11 @@ void select_app(GuiState &gui, const std::string &title_id) {
         LOG_ERROR("App with title id {} not found", title_id);
 }
 
-static constexpr ImU32 ARROW_COLOR = 0xFFFFFFFF; // White
-static float scroll_type, current_scroll_pos, max_scroll_pos;
-
 void draw_home_screen(GuiState &gui, EmuEnvState &emuenv) {
+    static constexpr ImU32 ARROW_COLOR = 0xFFFFFFFF; // White
+    static int scroll_type;
+    static float current_scroll_pos, max_scroll_pos;
```

Comment:
Looking at this functions, these two variables (current_scroll_pos, max_scroll_pos) are always set before being used, is there a reason they are static?

Reply:
yes, in version on this pr no need static